priam/src/main/java/com/netflix/priam/config/IConfiguration.java

```diff
@@ -809,7 +800,7 @@ default int getPostRestoreHookHeartbeatCheckFrequencyInMs() {
      * directory by Priam. These files live relative to the configuration directory.
      * @return A comma delimited list of relative file paths to the configuration directory
```

Comment:
nit: update the comment for return type.

Reply:
Done